### ISSUE TITLE
Duration option leads to NullPointerException

### ISSUE THREAD

**Issue by perotom** [tminglei/slick-pg#361]
I have to following model property:

`def totalTime = column[Option[Duration]]("total_time")`

SQL: `total_time INTERVAL`

which works perfectly when the database value is not null. As soon as the value is null I get the following exception:

```
[warn] c.z.h.p.ProxyConnection - db - Connection org.postgresql.jdbc.PgConnection@592d34d1 marked as broken because of SQLSTATE(08006), ErrorCode(0)
org.postgresql.util.PSQLException: Failed to create object for: interval.
	at org.postgresql.jdbc.PgConnection.getObject(PgConnection.java:586)
	at org.postgresql.jdbc.PgResultSet.getObject(PgResultSet.java:3360)
	at com.zaxxer.hikari.pool.HikariProxyResultSet.getObject(HikariProxyResultSet.java)
	at com.github.tminglei.slickpg.date.PgDateJdbcTypes$GenericDateJdbcType.getValue(PgDateJdbcTypes.scala:43)
	at slick.jdbc.SpecializedJdbcResultConverter$$anon$2.read(SpecializedJdbcResultConverters.scala:45)
	at slick.jdbc.SpecializedJdbcResultConverter$$anon$2.read(SpecializedJdbcResultConverters.scala:43)
	at slick.relational.ProductResultConverter.read(ResultConverter.scala:53)
	at slick.relational.ProductResultConverter.read(ResultConverter.scala:43)
	at slick.relational.TypeMappingResultConverter.read(ResultConverter.scala:133)
	at slick.jdbc.JdbcInvokerComponent$QueryInvokerImpl.extractValue(JdbcInvokerComponent.scala:36)
Caused by: java.lang.NullPointerException: null
	at org.postgresql.util.PGInterval.setValue(PGInterval.java:80)
	at org.postgresql.jdbc.PgConnection.getObject(PgConnection.java:571)
	at org.postgresql.jdbc.PgResultSet.getObject(PgResultSet.java:3360)
	at com.zaxxer.hikari.pool.HikariProxyResultSet.getObject(HikariProxyResultSet.java)
	at com.github.tminglei.slickpg.date.PgDateJdbcTypes$GenericDateJdbcType.getValue(PgDateJdbcTypes.scala:43)
	at slick.jdbc.SpecializedJdbcResultConverter$$anon$2.read(SpecializedJdbcResultConverters.scala:45)
	at slick.jdbc.SpecializedJdbcResultConverter$$anon$2.read(SpecializedJdbcResultConverters.scala:43)
	at slick.relational.ProductResultConverter.read(ResultConverter.scala:53)
	at slick.relational.ProductResultConverter.read(ResultConverter.scala:43)
	at slick.relational.TypeMappingResultConverter.read(ResultConverter.scala:133)
```

Looks like a problem with the Option. I am using version `15.2` of slick-pg with Play 2.6 and play-slick `3.0.0-M3`

**Comment by tminglei** [comment]
This issue should be fixed in #360 by @andrezimmermann.

**Comment by perotom** [comment]
Great work! Any planned release date? Very important bug fix.

**Comment by andrezimmermann** [comment]
You can workaround it using the 0.15-RC

**Comment by tminglei** [comment]
@perotom will release it ASAP.

**Comment by tminglei** [comment]
@perotom released slick-pg v0.15.3, pls update your dependency to use it.
